docs/fundamentals/code-analysis/overview.md

```diff
@@ -48,25 +45,25 @@ The following rules are enabled, by default, in .NET 5.0 Preview 8.
 | [CA2200](/visualstudio/code-quality/ca2200) | Usage | Warning | Rethrow to preserve stack details
 | [CA2247](/visualstudio/code-quality/ca2247) | Usage | Warning | Argument passed to TaskCompletionSource constructor should be <xref:System.Threading.Tasks.TaskCreationOptions> enum instead of <xref:System.Threading.Tasks.TaskContinuationOptions> |
 
-You can change the severity of these rules to disable them or elevate them to errors.
-
-For a full list of available code quality rules, see [Code quality rules](quality-rules/index.md).
+You can change the severity of these rules to disable them or elevate them to errors. You can also enable more rules.
 
-### Enable additional rules
```

Comment:
I believe this section might be linked from somewhere inside VS docs (not very sure though). I personally feel having this as a separate section is important, given so few CA rules are enabled by default and the most common question I get when moving to .NET analyzers is how to enable additional rules to match FxCop analyzers.